core/tests/protractor/collections.js

```diff
@@ -46,7 +46,32 @@ describe('Collections', function() {
     dropdown.element(by.css('.protractor-test-dashboard-link')).click();
     browser.waitForAngular();
     element(by.css('.protractor-test-create-activity')).click();
+    // Create new collection.
     element(by.css('.protractor-test-create-collection')).click();
+    // Add existing explorations.
+    element(by.css('.protractor-test-add-exploration-input')).sendKeys('0\n');
```

Comment:
A lot of this stuff is very low-level. Can we move it into a protractor_utils/collectionEditor.js file instead and have the stuff in this file be e.g. collectionEditor.addExploration('0')?